asapdiscovery-docking/asapdiscovery/docking/scripts/dock_small_scale_end_to_end.py

```diff
@@ -385,47 +392,17 @@ def main():
     #########################
 
     if args.postera:
-        import os
-
-        postera_api_key = os.getenv("POSTERA_API_KEY")
-
-        if postera_api_key is None:
-            raise ValueError("Environment variable POSTERA_API_KEY not found")
-
-        aws_access_key_id = os.getenv("AWS_ACCESS_KEY_ID")
-        if aws_access_key_id is None:
-            raise ValueError("Environment variable AWS_ACCESS_KEY_ID not found")
-
-        aws_secret_access_key = os.getenv("AWS_SECRET_ACCESS_KEY")
-        if aws_secret_access_key is None:
-            raise ValueError("Environment variable AWS_SECRET_ACCESS_KEY not found")
-
-        artifact_bucket_name = os.getenv("ARTIFACT_BUCKET_NAME")
-        if artifact_bucket_name is None:
-            raise ValueError("Environment variable ARTIFACT_BUCKET_NAME not found")
-
-        cloudfront_domain = os.getenv("CLOUDFRONT_DOMAIN")
-        if cloudfront_domain is None:
-            raise ValueError("Environment variable CLOUDFRONT_DOMAIN not found")
+        postera_settings = PosteraSettings()
```

Comment:
Do we want to check that all of the settings can be found earlier in the workflow so we fail fast by this point we have already started creating folders.

Reply:
Good point